tests/AutoConstructor.Tests/GeneratorTests.cs

```diff
@@ -857,6 +859,8 @@ internal partial class Test
         public readonly int _t2;
         protected readonly int _t3;
         private static readonly int _t4;
+        [field: AutoConstructorInject]
```

Comment:
```suggestion
        [AutoConstructorInject]
```
you can remove the `field` part (it is normally used to target the backing field of a property but this is already a field)